nautobot/docs/user-guides/graphql.md

```diff
@@ -131,17 +132,14 @@ query {
 ```
 
 This query will retrieve a list of all devices by their hostname.
-<div>
-  <details>
-    <summary>View GraphQL Query Results</summary>
-    <img src="../images/graphql/04-graphql-query-01.png">
-  </details>
-</div>
 
-<br />
+#### View GraphQL Query Results 1
+
+![View GraphQL Query Results 1](./images/graphql/04-graphql-query-01.png)
+
```

Comment:
Possibly restore the `<details>` HTML here and below as well?

Reply:
My bad I missed restoring these.